lock-api/src/main/java/com/palantir/lock/client/LockRefreshingRemoteLockService.java

```diff
@@ -131,4 +132,10 @@ public void dispose() {
         exec.shutdown();
         isClosed = true;
     }
+
+    @Override
+    public void close() throws IOException {
+        super.close();
+        dispose();
```

Comment:
I checked: yep. ExecutorService shutdown() is idempotent, as is writing the false to `isClosed`.